platforms-serverless/netlify-ci/test.sh

```diff
@@ -2,5 +2,6 @@
 
 set -eu
 
+# TODO This is not dynamic based on Napi env var, so should not even work... wtf?
```

Comment:
That's because we don't/can't set the ENV VAR in the CI. Thus it will always use the binary